examples/filesystem.py

```diff
@@ -69,20 +75,19 @@ def _iter(self, path):
             size=st.st_size,
             uid=st.st_uid,
             gid=st.st_gid,
-            ctime=datetime.fromtimestamp(st.st_ctime),
-            mtime=datetime.fromtimestamp(st.st_mtime),
-            atime=datetime.fromtimestamp(st.st_atime),
+            ctime=datetime.fromtimestamp(st.st_ctime, tz=ZoneInfo("UTC")),
+            mtime=datetime.fromtimestamp(st.st_mtime, tz=ZoneInfo("UTC")),
+            atime=datetime.fromtimestamp(st.st_atime, tz=ZoneInfo("UTC")),
             link=link,
         )
 
         if ifmt == stat.S_IFDIR:
-            for i in os.listdir(path):
+            for i in path.iterdir():
                 if i in (".", ".."):
                     continue
```

Comment:
the lines can be deleted now as we use Pathlib iterdir(), which doesn't return ".." and "." directories: